android/src/main/java/com/yubico/yubikit/android/transport/nfc/NfcSmartCardConnection.java

```diff
@@ -36,14 +38,16 @@ public class NfcSmartCardConnection implements SmartCardConnection {
      */
     private final IsoDep card;
 
+    private static final org.slf4j.Logger logger = LoggerFactory.getLogger(NfcSmartCardConnection.class);
```

Comment:
If you'd like, there's a [Lombok annotation](https://projectlombok.org/features/log) you can use to generate the `Logger` field (though it names it `log` by default). The main benefit of this in my opinion is that it eliminates the risk of passing the wrong class as the `getLogger()` argument. Which is admittedly a very small benefit if one isn't already using Lombok for something else. :slightly_smiling_face:

Reply:
Great suggestion, I verified the types manually for this PR. No plan for adding lombok atm.